odoo/fields.py

```diff
@@ -2049,7 +2078,13 @@ def _process_related(self, value):
 
 
 class Selection(Field):
-    """
+    """ Encapsulates an exclusive choice between different values.
+
+    The choices are specified as a list of tuples(`x`, `Y`) where
```

Comment:
isn't this is redundant with the `param selection` below?

Reply:
kind of, I found the Selection field worth a little comment, but in the end, it somehow says the same thing so I should just remove it.